bin/node/src/main.rs

```diff
@@ -112,6 +114,15 @@ fn main() -> sc_cli::Result<()> {
             .into()),
         None => {
             let runner = cli.create_runner(&cli.run)?;
+            if cli.aleph.experimental_pruning() {
+                warn!("Experimental_pruning was turned on. Usage of this flag can lead to misbehaviour, which can be punished. State pruning: {:?}; Blocks pruning: {:?};", 
```

Comment:
I love the idea of this log, but the things that get printed are hard to read – could you print them out in a more friendly format?

Reply:
I wanted to print to what value the arguments actually evaluate to. I can also print here the value of passed arguments.

Reply:
I meant it would be nice if you printed something representing what they are evaluated to, but without all the `Ok(Some....` boilerplate, more human-readable.

Reply:
Ok, done

Reply:
Uhm, that is definitely better, but I still wouldn't call it "human-readable". :P If making it more expressive is hard then it can stay as-is, but if you can make it write out the contents in an even nicer form that would be appreciated.

Reply:
But that would require digging inside substrate even more and in future might require update, if they change something :// anyway, only state-pruning with number is this ugly. Others are just enum names.